src/ess/bifrost/types.py

```diff
@@ -11,4 +11,3 @@
 FrameMonitor1 = reduce_t.Monitor2
 FrameMonitor2 = reduce_t.Monitor3
 FrameMonitor3 = reduce_t.Monitor4
```

Comment:
How do the aliases render in the graph display? If it is showing the underlying value this would be confusing?

Reply:
They do indeed show the underlying names.